.github/workflows/ci.yml

```diff
@@ -116,6 +116,10 @@ jobs:
         path: ${{ steps.pip-cache.outputs.dir }}
         restore-keys: |
             pip-ci-${{ runner.os }}-${{ matrix.pyver }}-${{ matrix.no-extensions }}-
+    - name: Install Docker Compose V2
+      if: ${{ matrix.os == 'ubuntu' }}
+      run: |
+        curl -L https://raw.githubusercontent.com/docker/compose-cli/main/scripts/install/install_linux.sh | sh
```

Comment:
It's discouraged to pipe scripts on the internet right into interpreters. Why didn't you use apt or pip? They are preferable.

Reply:
it's the only way that is mentioned in the [official docs](https://docs.docker.com/compose/cli-command/#install-on-linux).

Reply:
I just found out that the script specified in the documentation doesn't actually install docker compose v2 according to this [issue](https://github.com/docker/compose-cli/issues/1820#issuecomment-867659395).

Reply:
Sadly, the official docs don't always advise secure solutions.